radar_parlamentar/importadores/conv.py

```diff
@@ -89,14 +92,18 @@ def _gera_parlamentares(self):
                 parlamentar.save()
                 self.parlamentares[partido.nome].append(parlamentar)
 
-    def _gera_proposicao(self, num, descricao):
+    def _gera_proposicao(self, num, descricao, indexacao=None, ementa=None):
         prop = models.Proposicao()
         prop.id_prop = num
         prop.sigla = 'PL'
         prop.numero = num
-        prop.ementa = descricao
         prop.descricao = descricao
         prop.casa_legislativa = self.casa
+        if (indexacao and ementa) is not None:
```

Comment:
Essa sintaxe está incorreta em python.

`if (indexacao and ementa) is not None:` é diferente de `if indexacao is not None and ementa is not None:`.

Aliás, não sei se os dois precisam se tratados de forma conjunta....
Porque não um `if/else` para cada, individualmente?

Reply:
A ideia era que só estou trabalhando com dois casos: 
1. Quando se passa valor pra indexacao e ementa, ou seja, de forma conjunta
2. Quando nao se passa, ai só ementa é tratada e de forma diferente.

Talvez trabalhar de forma individual, seja mais expansivo, mas não sei se é necessário. E também o código fica um pouco mais enxuto. O que você acha?

Reply:
@peddrro entendi o seu ponto.
A questão é: O método possui os dois argumentos, e de forma "desvinculada".
Se alguém passar "ementa" e não passar "indexação", vai esperar que a "ementa" seja usada - e vice-versa.
Isso talvez gere dúvidas em quem for utilizar o método.
@leonardofl o que você acha?

Reply:
@diraol Verdade, que tal eu passar um parametro só entao, que represente os dois para não haver essa "desvinculação" e consequentemente não gerar dúvidas? Acha que seria uma boa?

Reply:
Pode ser uma boa opção. Aí vc cria esse objeto com um construtor que recebe os dois parâmetros (sem valor default), pra dizer que se criar o objeto, tem que usar os dois parâmetros.

Pra não criar uma classe nova, outra opção é criar dois métodos... um que não recebe (indexacao, ementa) e outro que recebe (indexacao, ementa), sem valores defaults. É um tanto quanto "javístico", mas acho q nesse caso tb ajudaria o entendimento de quem for usar o método.